src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/OidEnumerator.cs

```diff
@@ -16,26 +14,15 @@ internal OidEnumerator(OidCollection oids)
             _current = -1;
         }
 
-        public Oid Current
-        {
-            get
-            {
-                return _oids[_current];
-            }
-        }
+        public Oid Current => _oids[_current];
 
-        Object IEnumerator.Current
-        {
-            get
-            {
-                return _oids[_current];
-            }
-        }
+        object IEnumerator.Current => Current;
 
         public bool MoveNext()
         {
             if (_current == _oids.Count - 1)
```

Comment:
And, how about making this `>=`, then there's more justification for editing this file :smile: